src/app/3d/qgs3dmapcanvas.h

```diff
@@ -172,6 +185,8 @@ class Qgs3DMapCanvas : public QWidget
     QgsTemporalController *mTemporalController = nullptr;
 
     QSplitter *mSplitter = nullptr;
+
+    Qgs3DAxis *m3DAxis = nullptr;
```

Comment:
This should go into Qgs3DMapScene alongside other 3D object like the skybox

Reply:
Ok I will move it.